packages/components/lib/components.ts

```diff
@@ -1,4 +1,5 @@
 import Button from "./Button/Button";
 import Alert from "./Alert/Alert";
+import InputField from "./InputField/InputField";
 
-export { Button, Alert };
+export { Button, Alert, InputField };
```

Comment:
```suggestion
export { Button, Alert, InputField, InputFieldVariantEnum };
```

You should add here `InputFieldVariantEnum` because in Client i see that this const is using not only in this design component, so if you do not pass it here than you will cannot use it in Client from package

Reply:
thanks, already fixed that